### PR TITLE
fix: field blueprint filters

### PR DESCRIPTION
**Pull request by jacobmanderson**
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] Jira story/task is put into PR title
- [ ] Code has been self-reviewed
- [ ] Hard-to-understand areas have been commented
- [ ] Documentation has been updated
- [ ] My changes generate no new warnings
- [ ] Unit tests have been added for base functionality and known edge cases
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by Sung96kim — APPROVED** [review]
feels a little bad removing the stuff that people can filter by for field bps

**Comment by jacobmanderson** [comment]
> feels a little bad removing the stuff that people can filter by for field bps

i agree, but we don't really have a nice way of getting both . if we really want it i can probably take some time to look into it.

**Comment by Sung96kim** [comment]
I think at a minimum maybe docstring somewhere

File: indico/filters/field_blueprints.py [hunk 1]
```diff
@@ -1,43 +1,58 @@
...
-        uid (str): Filter by unique identifier
-        name (str): Filter by name
-        id (int): Filter by ID
-        task_type (str): Filter by task type
-        enabled (bool): Filter by enabled status
-        tags (str | List[str]): Filter by tags
+    :param op: The operator to apply. Use `FieldBlueprintFilter.LogicalOp` or `FieldBlueprintFilter.ComparisonOp` constants.
+    :param filters: A list of `FieldBlueprintFilter` instances. Required if `op` is LogicalOp.
+    :param field: The database field path to filter on. Use `FieldBlueprintFilter.Field` constants.
+    :param value: The value to filter by. Required if `op` is ComparisonOp.
```
Comment (Sung96kim) [inline indico/filters/field_blueprints.py]: nit: docstring format 👀
